index.js

```diff
@@ -12,9 +12,11 @@ var db = new Datastore({ filename: 'gb3.db', autoload: true });
 db.ensureIndex({ fieldName: 'content', unique: true }, function (err) {
 });
 
-var client = new irc.Client(process.env.SERVER, process.env.NICK, {
+const NICK = process.env.NICK;
```

Comment:
Maybe it would be good to use `process.env.NICK` but have a fallback to `gb3` or something if it's not given

Reply:
I ended up fixing it in a better way.